tree/treeplayer/src/TTreeProcessorMT.cxx

```diff
@@ -426,10 +446,12 @@ std::vector<std::string> TTreeProcessorMT::FindTreeNames()
 ///            for a TTree key in the file and will use the first one it finds.
 /// \param[in] nThreads Number of threads to create in the underlying thread-pool. The semantics of this argument are
 ///                     the same as for TThreadExecutor.
-TTreeProcessorMT::TTreeProcessorMT(std::string_view filename, std::string_view treename, UInt_t nThreads)
+/// \param[in] globalRange Struct containg {the first (inclusive), the last (exclusive)} global entry range
```

Comment:
I mentioned somewhere else that these docstrings have a typo ("containg") and the `{}` makes the sentence read as "Struct containing the first and the last global entry range", which is of course wrong. Maybe better:

```suggestion
/// \param[in] globalRange Global entry range to process, {begin (inclusive), end (exclusive)}.
```